cocli/cmd/cotsCorimCreate.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2021 Contributors to the Veraison project.
```

Comment:
Please align the name of the files with the associated commands, as per the existing convention of the comid and other files.